keyboards/40percentclub/tomato/keymaps/default/config.h

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Need a license header and an include guard. 

Same with the rest of the config.h files

Reply:
Whose name should I put in the license header though? The keyboard maker's? Mine?

Reply:
If unsure, then just `QMK` is fine.